backend/Virkarekisteri/src/Functions/Positions/CreatePositionsFromCsv.cs

```diff
@@ -0,0 +1,154 @@
+using System.Globalization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Virkarekisteri.Models;
+using Virkarekisteri.Repositories;
+
+namespace Virkarekisteri.Functions.Positions;
+
+public class CreatePositionsFromCsv(
+    ILogger<CreatePositionsFromCsv> logger,
+    IPositionRepository positionRepository,
+    IPositionNameRepository positionNameRepository
+)
+{
+    [Function("CreatePositionsFromCsv")]
+    public async Task<IActionResult> Run(
```

Comment:
This function will need a role authorization attribute (admin?)

Reply:
Added requiresEditRole which was in CreatePosition as well